File: comms/push-notifications/users/fetch-user/fetch-user.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.push_notifications.users.fetch(user_id: "comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8")

File: comms/push-notifications/users/fetch-user/fetch-user.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->pushNotifications->users->fetch(
    'comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8',
);
